net/http/authz/authorizer.go

```diff
@@ -16,19 +16,28 @@ import (
 
 var ErrNotAuthorized = errors.New("not authorized")
 
+var builtinGrants []*Grant // initialized in loopback_authz.go
+
 type Authorizer struct {
 	raftDB        *raft.Service
 	raftPrefix    string
 	policyByRoute map[string][]string
+	extraGrants   map[string][]*Grant
```

Comment:
🚲 🏠 : I'm not crazy about this name, can we pick something more descriptive? (`builtinGrants` seems good, actually...)